docs/docs/install-manuals/requirements.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: requirements
+title: System Requirements
+sidebar_position: 1
+---
+
+Since the microservice reference architecture consists of multiple microservices, a performant PC or Server is required to run all services.
+
+### Hardware requirements
```

Comment:
Maybe we should add a note that this is only an estimate and that we cannot guarantee anything. Just to be on the safe side.

Reply:
I mean we know that with the current amount of microservices it runs on the docker setup with 16 GB of RAM. I have checked the docker disk usage, which is around 7GB. I have left out the CPU as I have no idea there.
The only thing for which I also do not know the requirements would be a lokal kubernetes setup.

Reply:
Okay, fine for me then.